src/app/component/in-sight-side-bar/in-sight-side-bar.component.ts

```diff
@@ -316,6 +317,12 @@ export class InsightSideBarComponent implements OnInit {
     this.raiseTelemetry('copyToClipboard')
   }
 
+  checkLeaderboardData(event: boolean) {
+    if (event) {
+      this.isLeaderboardExist = true
```

Comment:
Please use event variable instead of true

Reply:
Ok, Updated.